services/blockchain-connector/app.js

```diff
@@ -68,12 +64,6 @@ nodeStatus.waitForNode().then(async () => {
 	app.run()
 		.then(async () => {
 			await init();
-
-			const schemas = await getSchemas();
-			setSchemas(schemas);
-
-			// Download the genesis block, if applicable
-			await getGenesisBlock();
```

Comment:
Don't we need to download genesis block during start if applicable?

Reply:
I've moved all the initialization logic within the `init` method in `services/blockchain-connector/shared/sdk/index.js`.